pyproject.toml

```diff
@@ -7,46 +7,43 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "sunkit_magex"
-description = "Potential field source surface modelling"
-readme = "README.rst"
-requires-python = ">=3.9"
-license = { file = "licenses/LICENSE.rst", content-type = "text/plain" }
+description = "Solar Magnetic field Extrapollation"
+requires-python = ">=3.10"
+readme = { file = "README.rst", content-type = "text/x-rst" }
+license = { file = "licenses/LICENSE.rst"}
```

Comment:
*muffled screaming*
```suggestion
license = { file = "licenses/LICENSE.rst" content-type = "text/plain"}
```

Reply:
This is no valid.

Reply:
Hmm needs more investigation then, but again needs fixing *not here*